erpnext/agriculture/doctype/destroy_plant_batch/destroy_plant_batch.json

```diff
@@ -0,0 +1,84 @@
+{
+ "creation": "2020-09-16 16:25:55.866077",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "plant_batch",
+  "destroy_count",
+  "reason_note",
+  "actual_date",
+  "amended_from"
+ ],
+ "fields": [
+  {
+   "fieldname": "plant_batch",
+   "fieldtype": "Link",
+   "label": "Plant Batch",
+   "options": "Plant Batch"
+  },
+  {
+   "fieldname": "destroy_count",
+   "fieldtype": "Int",
+   "label": "Destroy Count"
+  },
+  {
+   "fieldname": "actual_date",
+   "fieldtype": "Date",
+   "label": "Actual Date"
+  },
+  {
+   "fieldname": "amended_from",
+   "fieldtype": "Link",
+   "label": "Amended From",
+   "no_copy": 1,
+   "options": "Destroy Plant Batch",
+   "print_hide": 1,
+   "read_only": 1
+  },
+  {
+   "fieldname": "reason_note",
+   "fieldtype": "Data",
+   "label": "Reason Note"
+  }
+ ],
+ "is_submittable": 1,
+ "modified": "2020-09-16 14:25:25.677135",
+ "modified_by": "Administrator",
+ "module": "Agriculture",
+ "name": "Destroy Plant Batch",
```

Comment:
Talk to @neilLasrado about the name. I'm not sure about this one.

Reply:
ya sure